ztp/siteconfig-generator/siteConfig/siteConfigBuilder_test.go

```diff
@@ -1071,6 +1071,7 @@ func Test_filterExtraManifests(t *testing.T) {
   exclude: %s 
   include: %s
 `
+	const testSourceCrPath = "./testdata/source-cr-extra-manifest-copy/"
```

Comment:
I don't see any of the changes in this file in the original commit PR 1627. Can you please verify this backport is limited to the necessary changes.

Reply:
Unfortunately in the 4.13 branch the CI tests here depend on the hard coded list of files in the source-crs path

After discussing with Nahian we found that this fix, to use separate testdata files, was made in the main branch some time ago and decoupled the test result from the file list in source crs

The CI will not pass without updating these tests in 4.13